pynamodb/attributes.py

```diff
@@ -979,6 +979,28 @@ def as_dict(self):
             result[key] = value.as_dict() if isinstance(value, MapAttribute) else value
         return result
 
+    # some of the dict API
+
+    def get(self, item: _KT, default: Any = None) -> Union[_VT, None]:
+        if not self._is_attribute_container():
+            raise TypeError("MapAttribute must be acting as attribute container to MapAttribute.get")
+        return self.attribute_values.get(item, default)
+
+    def items(self):
+        if not self._is_attribute_container():
+            raise TypeError("MapAttribute must be acting as attribute container to MapAttribute.items")
+        return self.attribute_values.items()
+
+    def keys(self):
```

Comment:
```suggestion
    def keys(self) -> KeysView[_KT]:
```

Reply:
I added hinting to those methods, but it seems mypy was unhappy with `_KT` matching `str`: https://github.com/pynamodb/PynamoDB/pull/917/checks?check_run_id=2086230923

Any ideas? I'm not that familiar with mypy.

Reply:
I was also surprised by there being a `_KT` parameter, since I thought it's always `str`. Have to look into this.

Reply:
Should I use `str` instead of `_KT`? That would fix the mypy tests but did not seem right to me.